test/unit/src/utils/safeMath.test.ts

```diff
@@ -0,0 +1,111 @@
+import { MAX_INTEGER } from '@ethereumjs/util'
+import { ShardeumFlags } from '../../../../src/shardeum/shardeumFlags'
+import { SafeBalance } from '../../../../src/utils/safeMath'
+
+describe('SafeBalance', () => {
+    beforeEach(() => {
+        ShardeumFlags.unifiedAccountBalanceEnabled = false
```

Comment:
we should figure out a better way to do this for tests other than editing this global shared.

maybe just a jest mock of it?